plot_test.go

```diff
@@ -57,10 +57,10 @@ func TestLegendAlignment(t *testing.T) {
 		},
 		&recorder.Fill{
 			Path: vg.Path{
-				{Type: vg.MoveComp, X: 80, Y: 30},
-				{Type: vg.LineComp, X: 80, Y: 40},
-				{Type: vg.LineComp, X: 100, Y: 40},
-				{Type: vg.LineComp, X: 100, Y: 30},
+				{Type: vg.MoveComp, Pos: vg.Point{X: 80, Y: 30}},
```

Comment:
note that:
```go
{Type: vg.MoveComp, X: 80, Y: 30},
```

doesn't need to be as verbose as:
```go
{Type: vg.MoveComp, Pos: vg.Point{X: 80, Y: 30}},
```

(it's just my script that rewrote it like that)
it could be written as:
```go
{Type: vg.MoveComp, Pos: vg.Point{80, 30}},
```

which isn't that much verbose than:
```go
{Type: vg.MoveComp, Pos: vg.Pt(80, 30)},
```

is it?
I distinctly remember removing that function during the `vg` -> `vg/draw` move :)